app/views/comments/_comment.html.haml

```diff
@@ -1,8 +1,9 @@
 %p
   %strong Commenter:
-  = comment.commenter
+  = comment.user.email
 %p
   %strong Comment:
   = comment.body
 %p
-  = link_to 'Destroy Comment', [comment.article, comment], method: :delete, data: { confirm: 'Are you sure?' }
\ No newline at end of file
+  - if user_signed_in?
```

Comment:
should use the can? stuff everywhere you can.

should be
```
if can? :destroy, comment
```
but I would test that